src/clicache/src/CacheFactory.cpp

```diff
@@ -84,6 +85,12 @@ namespace Apache
 					//TODO::split
           SafeConvertClassGeneric::SetAppDomainEnabled(appDomainEnable);
 
+          if (appDomainEnable)
+          {
+            // Register managed AppDomain context with unmanaged.
+            apache::geode::client::createAppDomainContext = &Apache::Geode::Client::createAppDomainContext;
```

Comment:
No, it just looked better here since the surrounding code was doing things with the appDomainEnabled flag. Where it came from had no other AppDomain code and didn't check the flag so it was always "enabled". Now it will be enable only when the `appdomain-enabled` property is set.